cypress/integration/login.spec.ts

```diff
@@ -0,0 +1,54 @@
+// Just a simple test to make sure Cypress is running correctly
+// For more info, @see https://on.cypress.io/introduction-to-cypress
+
+import { emulatorsResetFirebase } from "cypress/support/util/db";
+import {
+  getButton,
+  getByBem,
+  getByText,
+  getInput,
+} from "cypress/support/util/get";
+import { visitSpaceInside } from "cypress/support/util/visit";
+
+type LogIn = (options: {
+  worldSlug: string;
+  spaceSlug: string;
+  email: string;
+  pass: string;
+}) => void;
+
+const logIn: LogIn = ({ worldSlug, spaceSlug, email, pass }) => {
+  visitSpaceInside({ worldSlug, spaceSlug });
+  getByText("Log In").click();
+  getInput("email").type(email);
+  getInput("password").type(pass);
+  getButton("Log in").click();
+  getByBem("AttendeeLayout__main").should("exist");
+};
+
+describe("plain attendee login using pre-seeded DB", () => {
+  // these should be pre-seeded into DB
+  const worldSlug = "world-01";
+  const spaceSlug = "space-01";
+  const email = "user-01@some.email";
+  const pass = "user-01-some-password";
```

Comment:
[Suggestion] let's have a config/constant file for these things somewhere

Reply:
Not for tests. They are anti-DRY by nature.

Most tests should have easy to read hard-coded values in `beforeEach()` or inside `it()` or even inside `describe()` so that it's KISS and self-contained. Unlike code that runs the project, tests will have many repeatable constants created for the purpose of a spec and removed just after it so it makes no sense to keep them elsewhere.

The above being said, these few might be put in another file simply because they're part of the pre-seed so should be available for every test, but for now there are just a token of 2 specs with 2 tests, just so we get the pipeline going, not needing to spend time on premature optimization.